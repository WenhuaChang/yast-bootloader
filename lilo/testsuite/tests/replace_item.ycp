{
    include "testsuite.ycp";
    include "lilo/routines.ycp";

    TEST(``(replaceitem([1,2,3,4,5], "v", 2)), [], nil);
    TEST(``(replaceitem([1,2,3,4,5], "v", 0)), [], nil);
    TEST(``(replaceitem([], "v", 1)), [], nil);
    TEST(``(replaceitem([1], "v", 1)), [], nil);
    TEST(``(replaceitem([], "v", 0)), [], nil);

}
