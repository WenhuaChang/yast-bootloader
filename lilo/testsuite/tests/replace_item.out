Return	[1, 2, "v", 4, 5]
Return	["v", 2, 3, 4, 5]
Return	["v"]
Return	[1, "v"]
Return	["v"]
